docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/downloading-google-cloud-storage-loader/index.md

```diff
@@ -4,43 +4,48 @@ date: "2020-03-02"
 sidebar_position: 0
 ---
 
+```mdx-code-block
+import {versions} from '@site/src/componentVersions';
+import CodeBlock from '@theme/CodeBlock';
+```
+
 Docker images can be found on [Docker Hub](https://hub.docker.com/r/snowplow/snowplow-google-cloud-storage-loader).
 
 Loader can be run with:
 
-```docker
-docker run \
-  -v $PWD/config:/snowplow/config \
-  -e GOOGLE_APPLICATION_CREDENTIALS=/snowplow/config/credentials.json \ # if running outside GCP
-  snowplow/snowplow-google-cloud-storage-loader:0.5.1 \
-  --runner=DataFlowRunner \
-  --jobName=[JOB-NAME] \
-  --project=[PROJECT] \
-  --streaming=true \
-  --workerZone=[ZONE] \
-  --inputSubscription=projects/[PROJECT]/subscriptions/[SUBSCRIPTION] \
-  --outputDirectory=gs://[BUCKET] \
-  --outputFilenamePrefix=output \ # optional
-  --shardTemplate=-W-P-SSSSS-of-NNNNN \ # optional
-  --outputFilenameSuffix=.txt \ # optional
-  --windowDuration=5 \ # optional, in minutes
-  --compression=none \ # optional, gzip, bz2 or none
-  --numShards=1 \ # optional
-  --dateFormat=YYYY/MM/dd/HH/ \ # optional
-  --labels={\"label\": \"value\"} \ #OPTIONAL
+<CodeBlock language="bash">{
```

Comment:
@istreeter - I changed these code blocks from docker code blocks -> bash code blocks after you approved the PR. Does that still seem reasonable?

Reply:
Very reasonable!

I'll be completely honest, I haven't looked up what this looks like.  But bash sounds to me like the sensible choice.

I reckon you're good to merge.